nailgun/entities.py

```diff
@@ -4966,6 +4969,28 @@ def __init__(self, server_config=None, **kwargs):
         super(PuppetModule, self).__init__(server_config, **kwargs)
 
 
+class CompliancePolicies(Entity, EntityReadMixin):
+    """A representation of a Policy entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'location': entity_fields.OneToManyField(Location),
+            'name': entity_fields.StringField(
+                required=True,
+                str_type='alpha',
+                length=(4, 30),
+                unique=True
+            ),
+            'organization': entity_fields.OneToManyField(Organization),
+            'hosts': entity_fields.ListField()
+        }
```

Comment:
The compliance Policies has other fields as well, If you add them now it would be helpful. Just a suggestion.

Reply:
I'll keep it in my mind, I just work on something else now, I'm sorry.